examples/controlnet/train_controlnet_flax.py

```diff
@@ -806,6 +812,17 @@ def main():
     validation_rng, train_rngs = jax.random.split(rng)
     train_rngs = jax.random.split(train_rngs, jax.local_device_count())
 
+    def compute_snr(timesteps):
```

Comment:
Could you also add a comment on the reference like here?

https://github.com/huggingface/diffusers/blob/ce144d6dd05c4c588d9f2970301ace70eee16d5d/examples/text_to_image/train_text_to_image.py#L497